ai2thor/tests/test_unity.py

```diff
@@ -1004,6 +1005,32 @@ def test_get_reachable_positions(controller):
     except:
         pass
 
+# Test for Issue: 477
```

Comment:
Should probably run black on this file.

Reply:
That comment actually passed black, but i merged in main to get all the other reformatted code.